playbook/app/pb_kits/playbook/pb_dashboard_value/dashboard_value.rb

```diff
@@ -13,12 +13,24 @@ class DashboardValue < Playbook::KitBase
                         default: {}
 
       def formatted_stat_value
-        { **stat_value, value: stat_value[:value].to_i }
+        { **stat_value, value: sanitized_stat_value }
       end
 
       def classname
         generate_classname("pb_dashboard_value_kit", align)
       end
+
+      # rubocop:disable Lint/FloatComparison
```

Comment:
I'm the last person you'll find disabling a rubocop lint rule. But using `value.to_f == value.to_i` is the crux of the logic here. There is no way around it.